README.md

```diff
@@ -28,7 +34,7 @@ which will create a new json in the current dir based on the com.gog.Template.js
 
 You can then build and install it directly thus:
 
-`flatpak-builder --user --install build gen_com.gog.BaldursGate2EnhancedEdition.json --force-clean --arch=i386`
+`flatpak-builder --user --install build gen_com.gog.BaldursGate2EnhancedEdition.json --force-clean --arch=x86_64`
```

Comment:
I couldn't actually test this with `Baldur's Gate 2: Enhanced Edition` as I don't own the game.

Reply:
BG2EE is hit by #17 in any case; so currently broken in any case, sadly.